src/fonts/myfont.scss

```diff
@@ -0,0 +1,36 @@
+@import '~@angular/material/theming';
```

Comment:
Separate all fonts into their files. Ensure they are properly renamed. Comments and variable names should be also updated to reflect the same.